config/initializers/riiif.rb

```diff
@@ -0,0 +1,33 @@
+# Get files via HTTP (they're in fedora)
+Riiif::Image.file_resolver = Riiif::HTTPFileResolver.new
+
+# Tell RIIIF how to resolve the identifier to a URI in Fedora
+Riiif::Image.file_resolver.id_to_uri = lambda do |id|
+  ActiveFedora::Base.id_to_uri(CGI.unescape(id)).tap do |url|
+    logger.info "Riiif resolved #{id} to #{url}"
+  end
+end
+
+Riiif::Image.info_service = lambda do |id, _file|
+  # id will look like a path to a pcdm:file
+  # (e.g. rv042t299%2Ffiles%2F6d71677a-4f80-42f1-ae58-ed1063fd79c7)
+  # but we just want the id for the FileSet it's attached to.
+
+  # Capture everything before the first slash
+  fs_id = id.sub(/\A([^\/]*)\/.*/, '\1')
+  resp = ActiveFedora::SolrService.get("id:#{fs_id}")
+  doc = resp['response']['docs'].first
+  raise "Unable to find solr document with id:#{fs_id}" unless doc
+  { height: doc['height_is'], width: doc['width_is'] }
+end
+
+def logger
+  Rails.logger
+end
```

Comment:
Missed this before, but this isn't right. You are effectively defining a logger method at top-level ruby scope. Or somewhere else that probably isn't what we want. 

What was intended here? Did this actually come from a riiif-generated template? grrr.